e2e/tests/common.rs

```diff
@@ -0,0 +1,137 @@
+use ethcontract::web3::api::Web3;
```

Comment:
Just a suggestion, the common methods could be added to the `e2e` crate directly and then the tests would call into it. There is no real advantage to this, however, and just something that I thought would be "cleaner".

Reply:
I toke the suggestion of having a common submodule inside the tests from here: https://doc.rust-lang.org/rust-by-example/testing/integration_testing.html

But in our case we have an entire project dedicated to tests so having it inside src/ can make sense